contentcuration/contentcuration/templatetags/perseus_tags.py

```diff
@@ -1,15 +1,10 @@
-import re
+import json
 from django import template
 from django.template.defaultfilters import stringfilter
 
 register = template.Library()
 
 @register.filter
 @stringfilter
-def escape_newline(value):
-	return value.replace('\n', '\\n')
-
-@register.filter
-@stringfilter
 def escape_chars(value):
-	return value.replace('\n', '\\n').replace('\\"', '"')
\ No newline at end of file
+    return json.dumps(value)[1:-1]
```

Comment:
Or would it be easier to just leave off the quotes in the template file itself? Then you're injecting a JSON-serialized value (including its quotes) into a valid position embedded in the outer JSON file.